extensions/mssql/src/main.ts

```diff
@@ -200,7 +202,7 @@ async function handleOpenNotebookTask(profile: azdata.IConnectionProfile): Promi
 async function handleOpenClusterStatusNotebookTask(profile: azdata.IConnectionProfile, appContext: AppContext): Promise<void> {
 	const notebookRelativePath: string = 'notebooks/tsg/cluster-status.ipynb';
 	const notebookFullPath: string = path.join(appContext.extensionContext.extensionPath, notebookRelativePath);
-	if (!Utils.fileExists(notebookFullPath)) {
+	if (!(await promisify(exists)(notebookFullPath))) {
 		vscode.window.showErrorMessage(localize("fileNotFound", "Unable to find the file specified"));
```

Comment:
It looks like the current suggested way is to use fs.access or fs.stat (which both have promise versions in fs.promises) instead of fs.exists (which is deprecated)

[https://stackoverflow.com/a/4482701](https://stackoverflow.com/a/4482701)

Reply:
Well that would be annoying to change is most places since it throws if it doesn't exist. I'll look into changing these.